e2e/config/config.go

```diff
@@ -62,6 +62,16 @@ type PVCSpec struct {
 	AccessModes      string `json:"accessModes"`
 }
 
+// Deployer represents a deployer configuration.
+// The `Name` is used to identify the deployer in tests,
+// while `Type` specifies the type of deployer.
+// Three types are supported:
+// appset, subscr, and disapp.
```

Comment:
Lets rewrap the lines to the common line length. I think best place to document name and type is inside the struct.